### PR TITLE
update to 14.1 + aarch64

### PR DESCRIPTION
**Pull request by matthiasdiener**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -47,7 +47,7 @@ requirements:
 
 test:
   commands:
-    - gdb --version
+    - $PREFIX/bin/gdb --version
```
Comment (phil-blain) [inline recipe/meta.yaml:50]: is this needed because `gdb` is installed in the aarch64 image ? If so, can we mention it in the commit message ?

File: recipe/run_test.sh [hunk 2]
```diff
@@ -1,4 +1,6 @@
-#!/usr/bin/env bash -ex
+#!/bin/bash
+
+set -ex
```
Comment (phil-blain) [inline recipe/run_test.sh:3]: is this change needed ? is so, please mention why in the commit message.

**Review by phil-blain — COMMENTED** [review]
Hi @matthiasdiener, thanks for the PR! I left a few questions below.

File: recipe/run_test.sh (hunk 2 above)
Comment (matthiasdiener) [reply recipe/run_test.sh:3]: This change isn't really needed, I just did it to make the file follow the same header structure as `build.sh`.

**Review by phil-blain — APPROVED** [review]
Thanks, this looks good. 

Yes, you are welcome to help as a new maintainer, thanks!. I don't know if you'll automatically be added by the automation once the PR is merged, if not let me know and I'll add you manually.

**Comment by matthiasdiener** [comment]
Thank you! It did add me automatically to the automation as far as I can see.

**Comment by jakirkham** [comment]
2 of the Travis CI jobs died during the build on `main`. Tried to restart them. Would keep an eye on those in case the same issue reoccurs

**Comment by jakirkham** [comment]
One Travis CI job still died. Restarted again

Might be worth looking into cross-compilation for a more robust solution
